src/components/Icons.tsx

```diff
@@ -641,14 +670,14 @@ otc('EUR', 'OTC_SX5E');
 otc('French', 'OTC_FCHI');
 otc('German', 'OTC_GDAXI');
 otc('JPY', 'OTC_N225');
-otc('Spanish', 'OTC_IBEX35');
```

Comment:
@hamza-deriv this is used in tests, you have to leave it our remove it from tests too )